enterprise/server/hostedrunner/hostedrunner.go

```diff
@@ -178,6 +179,7 @@ func (r *runnerService) createAction(ctx context.Context, req *rnpb.RunRequest,
 				{Name: "container-image", Value: RunnerContainerImage},
 				{Name: "recycle-runner", Value: "true"},
 				{Name: "workload-isolation-type", Value: "firecracker"},
+				{Name: platform.WorkloadTypePropertyName, Value: "remote-bazel"},
```

Comment:
I would prefer to avoid introducing another platform property since these are used as the snapshot key and will make it harder to make a change that we want to do in the future, which is sharing snapshots between remote bazel and workflows.

Can we instead change the isWorkflow logic to isCIRunner and have it check `Command.Arguments[0] == "./buildbuddy_ci_runner"`? We do have some precedent for this [here](https://github.com/buildbuddy-io/buildbuddy/blob/a1845b6e9ee69aefe809f88b84f5d2abd29ea09d/enterprise/server/remote_execution/executor/executor.go#L122-L126) if you wanted to move that to a common location (maybe `platform.go`)

Reply:
Good idea - changed